keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaTaskMonitorAgent.kt

```diff
@@ -65,7 +65,10 @@ class OrcaTaskMonitorAgent(
                     // when we get not found exception from orca, we shouldn't try to get the status anymore
                     taskTrackingRepository.delete(it.id)
                   }
-                  else -> throw e
+                  else -> log.warn(
```

Comment:
I understand the goal here, but I don't think we can just ignore these errors as those tasks will remain "stuck". We should set the resource status to an error status and/or clean-up the task tracking record, IMO.

Reply:
agree - I think we need to have an additional mechanism for clean up tasks. we should address it in a different PR.

Reply:
I think this is strictly better than we have right now, so I'm inclined to get this out for now, and circle back to it.